Library/RSBot.Core/Kernel.cs

```diff
@@ -100,8 +100,8 @@ private static async Task ComponentUpdaterAsync()
                     SpawnManager.Update(elapsed);
 
                     // Collision stuffs
-                    var currentRegionId = Game.Player.Movement.Source.Region;
-                    if (CollisionManager.CenterRegionId != currentRegionId)
+                    var currentRegionId = Game.Player.Position.Region.Id;
+                    if (CollisionManager.CenterRegionId != currentRegionId && currentRegionId != 0)
```

Comment:
![image](https://user-images.githubusercontent.com/14323838/199486388-49e97cd9-e182-463e-b965-c2d3e98cdc8d.png)
You dont need use `Id` just use region like in photo.

Reply:
Also check the "RSBot.EnableCollisionDetection" config before call the CollisionManager.Update()
Because you are removed the config checking in CollisionManager.Update method.

Reply:
The check that I removed was duplicated. CollisionManager.Enabled is the same.